index.html

```diff
@@ -10,17 +10,36 @@
     <link href="https://fonts.googleapis.com/css2?family=Merriweather+Sans:ital,wght@0,300;0,400;1,300;1,400&family=Merriweather:ital,wght@0,300;0,400;1,300;1,400&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
     <link rel="stylesheet" href="styles.css">
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.0.2/build/global/luxon.min.js">
+    </script>
 </head>
 <body>
-
-    <section id="bookCont" class="container text-center p-1">
+    <header class="navbar-expand content-font rounded pb-5">
+        <nav class="navbar d-flex justify-content-between background-color border border-dark">
+            <a href="#" class="navbar-brand flex-row text-white">Awesome Books</a>
+            <ul id="nav-items" class="navbar-nav flex-row">
+                <li class="nav-item p-2">
+                    <button type="button" class="nav-link badge badge-light" data-target="bookCont">List</button>
+                </li>
+                <li class="nav-item p-2">
+                    <button type="button" class="nav-link badge badge-light" data-target="addBook">Add New</button>
+                </li>
+                <li class="nav-item p-2">
+                    <button type="button" class="nav-link badge badge-light" data-target="contact">Contact</button>
+                </li>
+            </ul>
+        </nav>
+        <div id="dateDisplay" class="navbar d-flex flex-row-reverse clear-background border border-dark">
+        </div>
+    </header>
+    <section id="bookCont" class="container text-center p-1"> 
         <div id="title"><h2 class="title-font">All awesome books</h2></div>
         <div id="bookList" class="">
-            <ul id="booksSection" class="d-flex flex-column d-flex align-items-center p-0 border border-dark bg-white rounded content-font">
+            <ul id="booksSection" class="d-flex flex-column d-flex align-items-center p-0 border border-dark clear-background rounded content-font">
             </ul>
         </div>
```

Comment:
- [ ] As the provided wireframe is, we should try to make our version look alike as much as we can, the footer is overriding the list section whereas on the original wireframe there are a margin between both hence that gives a bad user experience, 

|orginal wirframe|Your design version|
|---|---|
|![Screenshot from 2021-09-16 20-11-36](https://user-images.githubusercontent.com/55918152/133664481-6cbc0156-88a6-480c-8034-054e46b034e0.png)|![Screenshot from 2021-09-16 20-12-08](https://user-images.githubusercontent.com/55918152/133664510-7c2a4bf2-02b3-441e-ad93-a1c305600911.png)|

_Note: Please notice that the footer is overlapping your books list as it designed in the original wireframe_ => Would you address that?